docs/using-template-repos-for-assignments.md

```diff
@@ -0,0 +1,48 @@
+Importing starter code for assignments is a very useful feature and with GitHub recently shipping the template repositories, we can leverage them in Classroom to make importing starter code much easier and faster!
+
+### How to use template repositories for an assignment?
+
+#### Before you begin
+To use template repositories for assignments, you'll need the following:
+
+* Set-up a GitHub classroom
+* An exercise (a repository that you have access to, which contains documentation, starter code, tests, or anything else your students need to begin work on an assignment).
```

Comment:
If we keep this, this shouldn't be called an exercise since we use this word to refer to assignments. An exercise repo would be OK.

Reply:
I've taken this from here: https://classroom.github.com/help/create-group-assignments
To keep the language consistent, should we make this change in that doc as well?

Reply:
How about,

> An exercise repository that you have access to, which contains documentation, starter code, tests, or anything else your students need to begin work on an assignment.